packages/mgt-components/src/styles/shared-sass-variables.scss

```diff
@@ -64,20 +64,27 @@ $text__color__main--light: $black;
 $text__color__sub1--light: $gray160;
 $text__color__sub2--light: $gray120;
 $text__color__sub3--light: $gray100;
+$text__color__sub4--light: $gray130;
```

Comment:
@beth-panx, do these need to be updated somewhere?

Reply:
I don't think it will impact any other components if that's the question. We should document the common colors as part of the contribution guide.